src/occupancy_data/real_occupancy.rs

```diff
@@ -175,31 +153,31 @@ impl Display for LoadsCount {
     }
 }
 
-fn occupancy_to_load(occupancy: Occupancy) -> Load {
+fn ratio_to_occupancy(occupancy: OccupancyRatio) -> Occupancy {
     debug_assert!(occupancy <= 100);
     if occupancy <= 30 {
-        Load::Low
+        Occupancy::Low
     } else if occupancy <= 70 {
-        Load::Medium
+        Occupancy::Medium
     } else {
-        Load::High
+        Occupancy::High
     }
 }
 
-pub struct LoadsData {
-    per_vehicle_journey: BTreeMap<BaseVehicleJourneyIdx, VehicleJourneyLoads>,
+pub struct OccupancyData {
+    per_vehicle_journey: BTreeMap<BaseVehicleJourneyIdx, VehicleJourneyOccupancy>,
 }
 
-struct VehicleJourneyLoads {
+struct VehicleJourneyOccupancy {
```

Comment:
```suggestion
struct VehicleJourneyOccupancies {
```
as it contains more than one occupancy data